queries/period_transfers.sql

```diff
@@ -2,7 +2,7 @@ with
 -- For a configurable permanent version visit: https://dune.xyz/queries/469110
 relevant_batch_info as (
     select concat('0x', encode(solver_address, 'hex')) as solver,
-           sum(gas_price_gwei * gas_used) / 10 ^ 9     as eth_spent,
+           sum(gas_price_gwei * gas_used) * 10 ^ 9     as eth_spent,
```

Comment:
should this become wei_spent?

Reply:
Technically yes, but then we may have to refactor all the Test DB schemas if we start changing column names. Technically also the `amount` below should become `amount_wei`. I kinda thought the PR was big enough that we could do this stuff after.